.travis.yml

```diff
@@ -7,6 +7,9 @@ install:
     - pip install --only-binary=numpy,scipy
     - pip install -r requirements.txt
     - pip install -r test-requirements.txt
+script: flake8
+after_success:
```

Comment:
Can you move this after the `script` section? It doesn't matter, but I find it easier to read